api/v1/openapi.yaml

```diff
@@ -856,6 +856,9 @@ definitions:
       labels:
         description: Labels describing the identity
         "$ref": "#/definitions/Labels"
+      labelSHA256:
```

Comment:
Can you rename this field to `labelsSHA256`?

Reply:
Done